.gitignore

```diff
@@ -1,5 +1,8 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
+#Temporary Development TRansfer
+/src/components/pages/Gamemode/ignored/
```

Comment:
This is very irregular. This file should serve everyone that is working in the repo. If you created the files then don't commit them. This only serves a single person in this case. If a future team member wanted to use this naming then the gitignore would block them.